addons/web/static/src/search/properties_group_by_item/properties_group_by_item.xml

```diff
@@ -0,0 +1,42 @@
+<templates xml:space="preserve">
+    <t t-name="web.PropertiesGroupByItem" owl="1">
+        <Dropdown togglerClass="'o_menu_item' + (isActive ? ' selected' : '')" beforeOpen.bind="loadDefinition">
+            <t t-set-slot="toggler">
+                Properties
```

Comment:
no distinction between different field of type properties? Should it not be linked to the property String?